tests/e2e/specs/dashboard-samples/RecomendedExtentions.spec.ts

```diff
@@ -102,38 +104,51 @@ suite(`Check if recommended extensions installed for ${samples}`, async function
         test(`Open "Extensions" view section`, async function (): Promise<void> {
             Logger.debug(`ActivityBar().getViewControl('Extensions'))?.openView(): open Extensions view.`);
             extensionsView = await (await new ActivityBar().getViewControl('Extensions'))?.openView();
-            Logger.debug(`extensionsView?.getContent().getSections(): get current section.`);
-            [extensionSection] = await extensionsView?.getContent().getSections() as ExtensionsViewSection[];
         });
 
         test(`Wait until extensions starts installation`, async function (): Promise<void> {
             Logger.info(`Time for extensions installation TimeoutConstants.TS_COMMON_PLUGIN_TEST_TIMEOUT=${TimeoutConstants.TS_COMMON_PLUGIN_TEST_TIMEOUT}`);
             await driverHelper.wait(TimeoutConstants.TS_COMMON_PLUGIN_TEST_TIMEOUT);
+            browserTabsUtil.refreshPage();
+            await driverHelper.wait(TimeoutConstants.TS_OPEN_EDITOR_TIMEOUT);
```

Comment:
can we use  explicit wait, something like `await projectAndFileTests.waitWorkspaceReadinessForCheCodeEditor()` ?

Reply:
Thanks for the hint!
Fixed.

New successful run: https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/e2e/job/basic/job/typescript-tests/7652/console